src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((arr) => arr.split(':'))
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. In this case, `arr` is not descriptive of the data it represents. A better name would be something like `styleDeclaration`.